Pod/Classes/XExtensionItem.h

```diff
@@ -96,10 +140,51 @@ typedef UIImage *(^XExtensionItemThumbnailProvidingBlock)(CGSize suggestedSize,
  */
 @property (nonatomic, copy) NSDictionary *userInfo;
 
+/**
+ Add entries from a dictionary-serializable custom object to this paramter object’s `userInfo` dictionary.
+ 
+ @param dictionarySerializable Object whose entries should be added to this paramter object’s `userInfo` dictionary.
+ */
+- (void)addEntriesToUserInfo:(id <XExtensionItemDictionarySerializing>)dictionarySerializable;
+
+#pragma mark - Initialization
+
+/**
+ Initialize an item source from a URL.
+ 
+ @param URL (Required) URL to be shared.
+ 
+ @return New item source instance.
+ */
 - (instancetype)initWithURL:(NSURL *)URL;
+
+/**
+ Initialize an item source from a string.
```

Comment:
I suppose I feel like "text" is a better way of describing the actual content being posted (similar to why Apple calls `NSExtensionItem`'s properties `attributedTitle` and `attributedContentText`). But it's really not a big deal and string seems perfectly appropriate.